src/extensions.rs

```diff
@@ -7,16 +7,24 @@ use crate::errors::*;
 pub trait CommandExt {
     fn print_verbose(&self, verbose: bool);
     fn status_result(&self, status: ExitStatus) -> Result<(), CommandError>;
-    fn run(&mut self, verbose: bool) -> Result<(), CommandError>;
-    fn run_and_get_status(&mut self, verbose: bool) -> Result<ExitStatus, CommandError>;
+    fn run(&mut self, verbose: bool, silence_stdout: bool) -> Result<(), CommandError>;
+    fn run_and_get_status(
+        &mut self,
+        verbose: bool,
+        silence_stdout: bool,
+    ) -> Result<ExitStatus, CommandError>;
     fn run_and_get_stdout(&mut self, verbose: bool) -> Result<String, CommandError>;
     fn run_and_get_output(&mut self, verbose: bool) -> Result<std::process::Output, CommandError>;
 }
 
 impl CommandExt for Command {
     fn print_verbose(&self, verbose: bool) {
         if verbose {
-            println!("+ {:?}", self);
+            if let Some(cwd) = self.get_current_dir() {
+                println!("+ {:?} {:?}", cwd, self);
```

Comment:
`cwd` might be quite long, or have unusual characters. Maybe this should be quoted?

Reply:
I mean that the CWD might be something like....

`/home/ahuszagh/PlayOnLinux's virtual drives/amazon_kindle/drive_c/Program Files/Amazon/Kindle/Microsoft.VC90.CRT`

Which, if you notice, isn't exactly very easy to read . We might want to ellipse some characters or just quote the whole thing.